src/pixelator/cli/analysis.py

```diff
@@ -133,7 +134,7 @@ def analysis(
     compute_colocalization,
     use_full_bipartite,
     polarization_normalization,
-    polarization_binarization,
+    polarization_permutations,
```

Comment:
I would call this `polarization_n_permutations` to keep it consistent with colocalization.

Reply:
Good suggestion!